javav2/example_code/timestream/src/main/java/com/timestream/write/ListTables.java

```diff
@@ -0,0 +1,80 @@
+//snippet-sourcedescription:[ListTables.java demonstrates how to return a list of your tables.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Timestream]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[05-04-2022]
+//snippet-sourceauthor:[scmacdon- AWS]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.timestream.write;
+
+//snippet-start:[timestream.java2.list_tables.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.timestreamwrite.TimestreamWriteClient;
+import java.util.List;
+import software.amazon.awssdk.services.timestreamwrite.model.ListTablesRequest;
+import software.amazon.awssdk.services.timestreamwrite.model.ListTablesResponse;
+import software.amazon.awssdk.services.timestreamwrite.model.Table;
+import software.amazon.awssdk.services.timestreamwrite.model.TimestreamWriteException;
+import software.amazon.awssdk.services.timestreamwrite.paginators.ListTablesIterable;
+//snippet-end:[timestream.java2.list_tables.import]
+
+/**
+ * To run this Java V2 code example, ensure that you have set up your development environment, including your credentials.
+ *
+ * For information, see this documentation topic:

```

Comment:
For more information, see the following documentation:

Reply:
fixed